### ISSUE TITLE
Cross publish for Scala 2.12

### ISSUE THREAD

**Issue by calvinlfer** [mingchuno/aws-wrap#8]
Hello again @mingchuno 
Sorry to bother you again, I believe you haven't done a cross publish for Scala 2.12 and older Scala versions. 
Since you are using Bintray-SBT, I believe you need to do a `sbt "+publish"` in order to perform cross build publishing. 

Thanks a lot
Cal

**Comment by mingchuno** [comment]
oh yes, your are correct, let me try again

**Comment by mingchuno** [comment]
done. You can try again

**Comment by calvinlfer** [comment]
[Looks good](https://bintray.com/mingchuno/maven/aws-wrap/0.9.2#files/com/github/dwhjames)
🍻
